### PR TITLE
Functional Tests : Mail alerts module - Customer notifications - Enable/Disable product availability

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Mail alerts module - Customer notifications - Enable/Disable product availability
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/6981861309
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    this.pageTitle = 'Mail alerts';
+
+    this.messageSettingsUpdated = 'Settings updated successfully';
+
+    this.productAvailabilityCheckbox = (status: boolean) => `#MA_CUSTOMER_QTY_${status ? 'on' : 'off'}`;
+    this.submitCustomerNotifications = 'button[name="submitMailAlert"]';
+  }
+
+  /* Methods */
+
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/modules/psEmailAlerts/index.ts:71]: Please add JS doc

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  async setProductAvailabilityStatus(page: Page, status: boolean): Promise<void> {
+    return this.setChecked(page, this.productAvailabilityCheckbox(status), true);
+  }
+
+  async getProductAvailabilityStatus(page: Page): Promise<boolean> {
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/modules/psEmailAlerts/index.ts:87]: Please add JS doc

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    return this.isChecked(page, this.productAvailabilityCheckbox(true));
+  }
+
+  async saveFormCustomerNotifications(page: Page): Promise<string> {
```
Comment (nesrineabdmouleh) [inline tests/UI/pages/BO/modules/psEmailAlerts/index.ts:96]: Please add JS doc

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/pages/BO/modules/psEmailAlerts/index.ts:71]: Fixed

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/pages/BO/modules/psEmailAlerts/index.ts:87]: Fixed

File: tests/UI/pages/BO/modules/psEmailAlerts/index.ts (hunk 3 above)
Comment (Progi1984) [reply tests/UI/pages/BO/modules/psEmailAlerts/index.ts:96]: Fixed

**Review by boubkerbribri — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by nesrineabdmouleh** [comment]
Thank you @Progi1984

**Pull request merged**